lib/MessageBar/styles.scss

```diff
@@ -8,6 +8,7 @@
   z-index: 3;
   a {
     color: white;
+    text-decoration: underline;
```

Comment:
Have is the hover state? At the moment it won't do anything right?

Reply:
@kyleharper this is true, however the bug was in certain places the a tag did not have the underline. 

But we should have a defined hover state for clarity, I'll check what people want and add it in.